Ginger/GingerCore/ALM/QCCore.cs

```diff
@@ -80,7 +80,13 @@ public override ObservableList<Activity> GingerActivitiesRepo
 
         public override ObservableList<ExternalItemFieldBase> GetALMItemFields(BackgroundWorker bw, bool online, ALM_Common.DataContracts.ResourceType resourceType)
         {
-            return ImportFromQC.GetALMItemFields();
+            ObservableList<ExternalItemFieldBase> tempFieldsList = ImportFromQC.GetALMItemFields();
+            almItemFields = new ObservableList<ExternalItemFieldBase>();
+            foreach (ExternalItemFieldBase item in tempFieldsList)
+            {
+                almItemFields.Add((ExternalItemFieldBase)item.CreateCopy());
+            }
```

Comment:
seems like GetALMItemFields function is same for all QC/RQM/Rally no? can't we place it in ALMCore instead having it in each alm type core class?